portfolio/src/main/webapp/baking.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> Chef Megan</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Food Gallery:</h1>
+      <div class="macaron-thumbnails">
+          <p> Macarons: </p>
+          <img src="/images/appo_mac.jpg" height=250/>
+          <img src="/images/mocha_mac.jpg" height=250/>
+          <img src="/images/pep_mac.jpg" height=250/>
+          <img src="/images/raz_mac.jpg" height=250/>
+          <img src="/images/rv_mac.jpg" height=250/>
+
+          <p>Not macarons:</p>
+          <img src="/images/biscotti.jpg" height=250/>
+          <img src="/images/cheese_tart.jpg" height=250/>
+          <img src="/images/crepe_cake.jpg" height=250/>
+          <img src="/images/challah.jpg" height=250/>
+          <img src="/images/pizza.jpg" height=250/>
+          <img src="/images/pasta.jpg" height=250/>
```

Comment:
img tag ends with `>` only please remove the `/`
also please add an alt for all images

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback